pipelines/resources/edgeclusters/edgecluster-deploy-edgecluster.yaml

```diff
@@ -26,44 +23,20 @@ spec:
       default: "false"
   stepTemplate:
     env:
-      - name: WORKDIR
-        value: "/workspace/ztp/$(params.pipeline-name)"
-      - name: OUTPUTDIR
-        value: "/workspace/ztp/$(params.pipeline-name)/build/$(context.taskRun.name)"
-      - name: EDGECLUSTERS_CONFIG
-        value: $(params.edgeclusters-config)
-      - name: KUBECONFIG
-        value: "$(workspaces.ztp.path)/kubeconfig"
-      - name: DEPLOY_EDGECLUSTERS_DIR
-        value: "deploy-edgecluster"
-      - name: SHARED_DIR
-        value: "shared-utils"
       - name: MOCK
         value: $(params.mock)
   steps:
-    - name: render-edgeclusters
-      image: "$(params.ztp-container-image)"
-      imagePullPolicy: Always
-      script: |
-        #!/usr/bin/bash
-
-        if [[ "${MOCK}" == 'false' ]]; then
-          cd ${WORKDIR}/${DEPLOY_EDGECLUSTERS_DIR}
-          ./render_edgeclusters.sh
-          sleep 3
-        else
-          echo "Render Edge-cluster: Mock mode on"
-        fi
-
     - name: deploy-edgeclusters
       image: "$(params.ztp-container-image)"
       imagePullPolicy: Always
       script: |
         #!/usr/bin/bash
 
         if [[ "${MOCK}" == 'false' ]]; then
-          cd ${WORKDIR}/${DEPLOY_EDGECLUSTERS_DIR}
-          ./deploy.sh
+          ztp create clusters \
+          --config "$(params.edgeclusters-config)" \
+          --output "/workspace/ztp/$(params.pipeline-name)" \
+          --wait 2h30m
```

Comment:
any way we can get this param from the env? Maybe from what's configured in the pipeline/task itself?

Reply:
I checked the Tekton [documentation](https://tekton.dev/docs/pipelines/variables/) and didn't find anything.